cli/src/main/resources/hudson/cli/client/Messages.properties

```diff
@@ -2,11 +2,18 @@ CLI.Usage=Jenkins CLI\n\
   Usage: java -jar jenkins-cli.jar [-s URL] command [opts...] args...\n\
   Options:\n\
    -s URL       : the server URL (defaults to the JENKINS_URL env var)\n\
-   -i KEY       : SSH private key file used for authentication\n\
+   -http        : use a plain CLI protocol over HTTP(S) (the default; mutually exclusive with -ssh and -remoting)\n\
+   -ssh         : use SSH protocol (requires -user; SSH port must be open on server, and user must have registered a public key)\n\
+   -remoting    : use deprecated Remoting channel protocol (if enabled on server; for compatibility with legacy commands or command modes only)\n\
```

Comment:
Two _transports_. Traditionally the fallback from TCP to HTTP Duplex was hidden from users—not reported at a default log level, and not manually overridable. I am simply not making changes to that decision, as it applies only to a deprecated mode anyway.